src/test/java/org/jabref/logic/citationkeypattern/CitationKeyGeneratorTest.java

```diff
@@ -131,6 +130,15 @@ void crossrefAndInAuthorNames() {
                         database), DEFAULT_UNWANTED_CHARACTERS));
     }
 
+    @Test
+    void testLatexCommandsAreStrippedFromCitationKey() throws ParseException {
+        BibEntry entry = new BibEntry()
+                .withField(StandardField.TITLE, "Building \\mkbibquote{Community}");
+        String pattern = "[title]";
```

Comment:
Did you read the issue? Did your LLM fail? Which tool.did you use?

> [bibtexkey] - [fulltitle]

Reply:
Yes, I read the issue and understood the problem about LaTeX commands not being stripped correctly from fields used in filename generation.
I used an AI assistant to refine my understanding of the issue and help structure the PR. However, the final implementation and test case were written based on my own understanding. If there is any discrepancy, I take full responsibility for the work.
I used ChatGPT as a supporting tool to draft and clarify ideas.

Reply:
I quoted the most important part of the issue above. Please adapt the test case accordingly.

Reply:
Hey @koppor
 i guess i need to read the documentation and little more about this to work on this currently.

Reply:
> Hey @koppor
>  i guess i need to read the documentation and little more about this to work on this currently.

You need to change a single line in your test case. Instead of `[title]` use the pattern reported by the user

Reply:
Thanks @koppor , have updated the pattern reported by the user by  [bibtexkey] - [fulltitle]

Reply:
But i guess there is more to the issue

Reply:
Hey @koppor i have made some changes accordingly, but still the OpenRewrite and UnitTest error is remaining, i will be checking it once again by tomorrow. If you have any hint please provide it , Thanks :)

Reply:
<img width="997" alt="grafik" src="https://github.com/user-attachments/assets/6b124862-141a-4aa1-baca-77decd6e42d8">
Click on details to see the exact cause of the failure